Create upload.md

shntnu opened the pull request:
#3

ErinWeisbart commented:
@shntnu Can you take a close look at this (at your convenience) and let me know if it's what you had in mind?
Please note that:
- I abstracted all the `cellpainting-gallery` paths to `s3://cellpainting-gallery/${PROJECT_DIRECTORY}/${PROJECT_NESTING}` We don't really use these variable names in any of our public workflows, but it does match the folder structure/nomenclature in #2.
- 0. Set up AWS still needs to be written. I don't think that linking to the jump-cellpainting steps is sufficient as it hard codes `jump` in many places. I don't have a great grasp on credentials and permissions so I'm afraid I either need you to write this or talk me through it before I can write it.

shntnu commented:
> Can you take a close look at this (at your convenience) and let me know if it's what you had in mind? Please note that:

It looks great!

> * I abstracted all the `cellpainting-gallery` paths to `s3://cellpainting-gallery/${PROJECT_DIRECTORY}/${PROJECT_NESTING}` We don't really use these variable names in any of our public workflows, but it does match the folder structure/nomenclature in [Create folder_structure.md #2](https://github.com/broadinstitute/cellpainting-gallery/pull/2).

Perfect!

> * `0. Set up AWS` still needs to be written. I don't think that linking to the jump-cellpainting steps is sufficient as it hard codes `jump` in many places. I don't have a great grasp on credentials and permissions so I'm afraid I either need you to write this or talk me through it before I can write it.

Yep – still figuring out this bit. I've added a preface: _Note: At present, only a few individuals – those with access to instructions [here](https://github.com/jump-cellpainting/aws#creating-an-aws-role-to-access-s3-buckets) – have the ability to upload data to `s3://cellpainting-gallery`._

> * Do we need `--request-payer requester` and/or `--metadata-directive REPLACE` flags in commands? They are not currently there.

Hm, I think `--metadata-directive REPLACE` is needed but not `--request-payer requester`. Thanks for spotting it! I've added 

All this LGTM!

Feel free to ask questions, if not, please go ahead and merge
